src/platform/mhv/secondary-nav/tests/MhvSecondaryNavMenu.unit.spec.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import { expect } from 'chai';
+import { Provider } from 'react-redux';
+import MhvSecondaryNavMenu from '../components/MhvSecondaryNavMenu';
+
+const mockStore = ({ isFeatureEnabled = false } = {}) => ({
+  getState: () => ({
+    featureToggles: {
+      // eslint-disable-next-line camelcase
+      mhv_secondary_navigation_enabled: isFeatureEnabled,
+    },
+  }),
+  subscribe: () => {},
+  dispatch: () => {},
+});
+
+const testSecNavItems = [
+  {
+    title: 'My HealtheVet',
+    iconClass: 'fas fa-home',
+    href: '/my-health',
+  },
+  {
+    title: 'Appointments',
+    abbreviation: 'Appts',
+    iconClass: 'fas fa-calendar',
+    href: `/my-health/appointments`,
+  },
+  {
+    title: 'Messages',
+    iconClass: 'fas fa-comments',
+    href: `/my-health/secure-messages`,
+  },
+  {
+    title: 'Medications',
+    abbreviation: 'Meds',
+    iconClass: 'fas fa-prescription-bottle',
+    href: `/my-health/medications`,
+  },
+  {
+    title: 'Records',
+    iconClass: 'fas fa-file-medical',
+    href: `/my-health/medical-records`,
+  },
+];
+
+describe('MHV Secondary Navigation Menu Component', () => {
+  it('renders when the toggle is on', () => {
+    const mock = mockStore({ isFeatureEnabled: true });
+    const { getAllByRole } = render(
+      <Provider store={mock}>
+        <MhvSecondaryNavMenu items={testSecNavItems} />
+      </Provider>,
+    );
+    expect(testSecNavItems).to.not.be.empty;
+    const links = getAllByRole('link');
+    expect(links.length).to.eql(testSecNavItems.length);
+    testSecNavItems.forEach((navItem, i) => {
+      expect(links[i].pathname).to.be.eql(navItem.href);
+    });
+  });
+
+  it('does not render when the toggle is off', () => {
+    const mock = mockStore();
+    const { getAllByRole } = render(
+      <Provider store={mock}>
+        <MhvSecondaryNavMenu items={testSecNavItems} />
+      </Provider>,
+    );
+    expect(testSecNavItems).to.not.be.empty;
+    expect(() => getAllByRole('link')).to.throw();
```

Comment:
Testing that nothing is rendered is tricky. 

For this case, I would suggest:

```
const { container } = render(...);
expect(container).to.be.empty;

```

Reply:
Interesting. That was not obvious from looking at the contents of container while debugging, but I tested it with and without content and it works! Changed.